packages/mediacenter/kodi/scripts/systemd-addon-wrapper

```diff
@@ -33,3 +33,17 @@ if [ -f "/storage/.kodi/addons/$1/system.d/$1.service" ] ; then
     systemctl start "$1.service"
   fi
 fi
+
+if [ ! -d /storage/.cache/module-overlays ] ; then
+  mkdir -p /storage/.cache/module-overlays
+fi
```

Comment:
Maybe this if block (or even only _mkdir -p_) should be moved inside next if? This would leave base system more clean (no point of having empty folder for most users).